### PR TITLE
fix noResult showing on search when typing search text and change people photos ratio

### PR DESCRIPTION
**Pull request by KonradBauer**
Check this changes - type sth on people or movies input. Tiles is no more too much width.

### CODE REVIEW

File: src/features/Input/index.js [hunk 1]
```diff
@@ -47,16 +52,20 @@ export default () => {
       value: target.value.trim() ? target.value : undefined,
     });
 
-    dispatch(changeMoviesSearchText(""));
```
Comment (piotrpaczuski) [inline src/features/Input/index.js:50]: You unnecessarily moved this dispatch to setTimeout below.

File: src/features/Input/index.js [hunk 2]
```diff
@@ -29,16 +29,21 @@ export default () => {
       key: searchQueryParamsName,
       value: target.value.trim() ? target.value : undefined,
     });
-    dispatch(changePeopleSearchText(""));
```
Comment (piotrpaczuski) [inline src/features/Input/index.js:32]: You unnecessarily moved this dispatch to setTimeout below.

File: src/App.js [hunk 3]
```diff
@@ -56,6 +63,7 @@ export const App = () => {
     dispatch(fetchMovieDetailsAndCredits());
     dispatch(fetchPeople());
     dispatch(fetchPeopleDetails());
+    dispatch(fetchSearchPeople());
```
Comment (piotrpaczuski) [inline src/App.js]: This dispatch is unneccessary, because he was added in the other pull request.

File: src/App.js [hunk 4]
```diff
@@ -34,7 +40,8 @@ import {
 } from "./features/movies/searchMoviesSlice";
 import { selectSearchMoviesText } from "./features/movies/searchMoviesSlice";
 import Input from "./features/Input";
-import { selectSearchPeopleStatus } from "./features/peoples/searchPeopleSlice";
+import { fetchSearchPeople, selectSearchPeopleStatus } from "./features/peoples/searchPeopleSlice";
```
Comment (piotrpaczuski) [inline src/App.js]: From this import you can delete fetchSearchPeople.

**Review by piotrpaczuski — CHANGES_REQUESTED** [review]
Change things, which I wrote in comments to lines.

File: src/App.js (hunk 3 above)
Comment (KonradBauer) [reply src/App.js]: I fix requested changes, check it out

**Review by piotrpaczuski — APPROVED** [review]
After your changes everything works, when I mess up the link by adding the wrong id, an error page is shown. Well done.

**Review by florekdeveloper — APPROVED** [review]
Everything works well, great job :)

**Pull request merged**